src/modules/home/components/accommodation-card/accommodation-card.tsx

```diff
@@ -0,0 +1,44 @@
+import CardActions from '@mui/material/CardActions';
+import Card from '@mui/material/Card';
+import CardContent from '@mui/material/CardContent';
+import CardMedia from '@mui/material/CardMedia';
+import Typography from '@mui/material/Typography';
+import FavoriteBorderIcon from '@mui/icons-material/FavoriteBorder';
+import FavoriteIcon from '@mui/icons-material/Favorite';
+import IconButton from '@mui/material/IconButton';
+import StarRate from '@mui/icons-material/StarRate';
+import CardImage from '#/assets/images/The-Balmoral-Executive-View-Room-e1682260891619.jpg';
+import { useState } from 'react';
+import { truncateString } from '#/utils/truncate-string.util';
+import { styles } from './styles';
+import { pink } from '@mui/material/colors';
+
+export function AccommodationCard() {
+  const [addToWishlist, setAddToWishlist] = useState(true);
+  return (
+    <Card sx={styles.card}>
+      <IconButton aria-label="add to favorites" sx={styles.iconButtonBorder} onClick={() => setAddToWishlist(!addToWishlist)}>
+        {addToWishlist ? <FavoriteBorderIcon sx={{ color: 'white' }} /> : <FavoriteIcon sx={{ color: pink[500] }} />}
+      </IconButton>
+      <IconButton aria-label="add to favorites" sx={styles.iconButton} onClick={() => setAddToWishlist(!addToWishlist)}>
+        {!addToWishlist || <FavoriteIcon sx={{ color: 'GrayText' }} />}
+      </IconButton>
```

Comment:
Why two buttons?

Reply:
> Why two buttons?

I think he did it to have a slightly darker filled heart at the back and a white bordered non-filled heart at the front, to give the illusion of a white bordered heart filled in dark transparent color

Reply:
yeah Shahobiddin is right. I added two buttons to make it visible in any background color like black or white